models/meteora/meteora_trades.sql

```diff
@@ -0,0 +1,159 @@
+ {{
+  config(
+        schema = 'meteora',
+        alias = 'trades',
+        partition_by = ['block_month'],
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+        unique_key = ['tx_id', 'outer_instruction_index', 'inner_instruction_index', 'tx_index','block_month'],
```

Comment:
i notice the unique keys here are different than tensorswap. is that normal for solana? on evm chains for dex trades, we use a universal set of unique keys on all projects

Reply:
tensorswap is nft marketplace, meteora is dex